CourseStream/CourseStream/Program.cs

```diff
@@ -1,22 +1,29 @@
-
 using CourseStream;
 using CourseStream.Core;
+using CourseStream.Data;
 using CustomExceptions;
-using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.AddSingleton<CourseStreamService>();
+// Access configuration
+var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
+
+// Configure your DbContext
+builder.Services.AddDbContext<CourseStreamDbContext>(options =>
+	options.UseNpgsql(connectionString));
+builder.Services.AddScoped<CourseStreamService>();
```

Comment:
Maybe it makes sense to create an issue to create an interface for `CourseStreamService`?

Reply:
Please, do not fix this in scope of this PR

Reply:
I needed to do this, because in other way dockers didn't start